packages/conf-pkg-config/conf-pkg-config.1.1/opam

```diff
@@ -0,0 +1,34 @@
+opam-version: "1.2"
+maintainer: "unixjunkie@sdf.org"
+authors: ["Francois Berenger"]
+homepage: "http://www.freedesktop.org/wiki/Software/pkg-config/"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+dev-repo: "git://github.com/ocaml/opam-repository"
+license: "GPL"
+build: [
+  ["pkg-config" "--help"]
+]
+install: [
+  ["ln" "-s" "/usr/local/bin/pkgconf" "%{bin}%/pkg-config"] {os = "openbsd"}
+]
+remove: [
+  ["rm" "-f" "%{bin}%/pkg-config"] {os = "openbsd"}
+]
+post-messages: [
+  "conf-pkg-config: A symlink to /usr/local/bin/pkg-config has been installed in the OPAM bin directory (%{bin}%) on your PATH. This is necessary for correct operation." {os = "openbsd"}
```

Comment:
this message does not match the install directive

Reply:
> this message does not match the install directive

Fixed, and also reworded the commit message to be clearer.